CHANGELOG.md

```diff
@@ -19,6 +21,12 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Disable `PodSecurityPolicy` for Kubernetes >= v1.25. ([#352](https://github.com/giantswarm/nginx-ingress-controller-app/pull/352))
 
+### Important notes
+
+Please upgrade to any `v2.18.x` version before upgrading to this release or above since the controller image contained in there migrates your setup to the Lease API.
```

Comment:
Nice! I think we need to make sure this note also goes out into the announcemnt to customers